requirements/test_requirements.txt

```diff
@@ -1,4 +1,5 @@
 pytest
 mock
+httpx
```

Comment:
I didn't see this used - was it an initial implementation but removed later?

Reply:
It's required for the FastAPI TestClient, according to [FastAPI documentation](https://fastapi.tiangolo.com/tutorial/testing/#using-testclient)